src/Microsoft.AspNetCore.Rewrite/RewriteMiddleware.cs

```diff
@@ -97,6 +98,7 @@ public Task Invoke(HttpContext context)
                     default:
                         throw new ArgumentOutOfRangeException($"Invalid rule termination {rewriteContext.Result}");
                 }
+                _logger.RewriteSummary(currentUrl);
```

Comment:
It's possible this logger statement won't be produced if SkipRemainingRules is true. It's also possible that this summary is inaccurate. This should instead go inside the `IRule` implementations that actually rewrite the URL.